realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1697,7 +1698,7 @@ ColumnIndices updateSchemaCache(ColumnIndices[] globalCacheArray) {
                     map.put(clazz, columnInfo);
                 }
             } catch (RealmMigrationNeededException e) {
-                throw e;
+                throw e; // WTF??
```

Comment:
It probably was here just to show that a RealmMigrationNeededException can be thrown here in this block.